native/core/src/execution/planner.rs

```diff
@@ -884,7 +884,7 @@ impl PhysicalPlanner {
                     func_name,
                     fun_expr,
                     vec![left, right],
-                    data_type,
+                    Field::new("foo", data_type, true),
```

Comment:
There have been changes in DF to use Field rather than DataType. Comet does not use the field name.

Reply:
can we use name `not_used` or to put the comment like that field name is not used in Comet?

Reply:
I think the type is still used, so what about leaving a small comment instead?

Reply:
I updated this to use the function name as the return field name